class-instant-articles-post.php

```diff
@@ -685,6 +685,15 @@ public function to_instant_article() {
 			$this->instant_article->withStyle( 'default' );
 		}
 
+		if ( isset( $settings_publishing[ 'likes_on_media' ] ) ) {
```

Comment:
Please add a comment to make it explicit that these settings affect the outcome of `$transformer->transformString(...)`